tracer/test/Datadog.Trace.SourceGenerators.Tests/Datadog.Trace.SourceGenerators.Tests.csproj

```diff
@@ -1,10 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Generated\placeholder.json" />
+  </ItemGroup>
+
```

Comment:
I think this is leftover from before, no? 🤔 
```suggestion
```

Reply:
Yes, it was. Fixed in eeab78539a8f827bfb37947ff88a19596b0eaaa7